MapleServer2/PacketHandlers/Login/CharacterManagementHandler.cs

```diff
@@ -61,8 +61,8 @@ public void HandleCreate(LoginSession session, PacketReader packet)
         {
             byte gender = packet.ReadByte();
             //packet.ReadShort(); // const?
-            var jobCode = (Job)packet.ReadShort();
-            //int jobCode = packet.ReadShort();
+            // var jobCode = (Job)packet.ReadShort();
+            int jobCode = packet.ReadShort();
             string name = packet.ReadUnicodeString();
             var skinColor = packet.Read<SkinColor>();
```

Comment:
Since we changed jobCode to use an explicit datatype, we should make sure the other variables are also using explicit datatypes to be consistent.

Reply:
Which variables are you referring to here?

Reply:
Late response I know, but we have skinColor for example declared implicitly (var), while other variables here are declared explicitly.